robot-server/tests/integration/http_api/commands/test_home.tavern.yaml

```
@@ -0,0 +1,141 @@
+test_name: home standalone command
+
+marks:
+  - usefixtures:
+      - run_server
+
+stages:
+  - name: Delete all runs # Ensure there is no active run
+    request:
+      url: '{host:s}:{port:d}/runs'
+      method: GET
+    response:
+      status_code: 200
+      verify_response_with:
+        - function: 'tests.integration.fixtures:delete_all_runs'
+          extra_kwargs:
+            host: '{host:s}'
+            port: '{port:d}'
+  - name: issue home Command params all
+    request:
+      url: '{host:s}:{port:d}/commands'
+      method: POST
+      params:
+        waitUntilComplete: true
+      json:
+        data:
+          commandType: home
+          params:
+            axes:
+              - x
+              - 'y'
```

Is this intentionally in quotes, or should it be:

```suggestion
              - y
```

---

y was highlighting up like it might be reserved keyword?.... I changed it.